src/Gui/DlgSettingsNavigation.ui

```diff
@@ -24,89 +24,122 @@
+          </property>
+          <item>
+           <property name="text">
+            <string>Top left</string>
+           </property>
+          </item>
+          <item>
+           <property name="text">
+            <string>Top right</string>
+           </property>
+          </item>
+          <item>
+           <property name="text">
+            <string>Bottom left</string>
+           </property>
+          </item>
+          <item>
+           <property name="text">
+            <string>Bottom right</string>
+           </property>
+          </item>
+         </widget>
+        </item>
+        <item row="1" column="3">
+         <widget class="QLabel" name="cornerLabel">
+          <property name="text">
+           <string>Corner</string>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+          </property>
+         </widget>
+        </item>
+        <item row="1" column="0">
+         <widget class="QLabel" name="stepLabel">
+          <property name="text">
+           <string>Steps by turn</string>
+          </property>
+         </widget>
+        </item>
+        <item row="1" column="1">
+         <widget class="Gui::PrefSpinBox" name="prefStepByTurn">
+          <property name="toolTip">
+           <string>Number of steps by turn when using arrows (default = 8 : step angle = 360/8 = 45 deg)</string>
+          </property>
+          <property name="minimum">
+           <number>4</number>
+          </property>
+          <property name="maximum">
+           <number>36</number>
+          </property>
+          <property name="value">
+           <number>8</number>
+          </property>
+          <property name="prefEntry" stdset="0">
+           <cstring>NaviStepByTurn</cstring>
+          </property>
+          <property name="prefPath" stdset="0">
+           <cstring>View</cstring>
+          </property>
+         </widget>
         </item>
-        <item>
-         <property name="text">
-          <string>Top right</string>
-         </property>
+        <item row="1" column="2">
+         <spacer name="horizontalSpacer">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+          <property name="sizeType">
+           <enum>QSizePolicy::MinimumExpanding</enum>
+          </property>
+          <property name="sizeHint" stdset="0">
+           <size>
+            <width>91</width>
+            <height>20</height>
+           </size>
+          </property>
+         </spacer>
         </item>
-        <item>
-         <property name="text">
-          <string>Bottom left</string>
-         </property>
+        <item row="2" column="1">
+         <widget class="Gui::PrefSpinBox" name="prefCubeSize">
+          <property name="toolTip">
+           <string>Size of the navigation cube</string>
+          </property>
+          <property name="minimum">
```

Comment:
Spinning by one seems a bit small to me, it might be better to spin by five or ten each click.

Reply:
I've added ten to single step [L125](https://github.com/FreeCAD/FreeCAD/pull/4603/files#diff-cd7e65479ecf30d24f0da42072b605500ff01b38688564ce32a732de9ac80e07R125)